regression_test_script

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -e
+echo "Starting up ..."
+
+default_time_out=30
+default_image_name="opendap/hyrax:snapshot"
+default_endpoint="http://localhost:8080/opendap/"
+
+time_out="${default_time_out}"
+test_image_name="${default_image_name}"
+end_point="${default_endpoint}"
+netrc_file=""
+tests="1 2 3 4 5 6 7 8 9 10"
+
+while getopts "ho:i:e:n:t:" OPTION; do
+    case $OPTION in
+    h)
+        echo "-h help -o time_out -i image_name -e end_point_url -n netrc_file -t \"test1 test2 ...\""
+	exit 0
+        ;;
+    o)
+        time_out=$OPTARG
+        ;;
+    i)
+        test_image_name=$OPTARG
+        ;;
+    e)
+        end_point=$OPTARG
+        ;;
+    n)
+        netrc_file=$OPTARG
+        ;;
+    t)
+        tests=$OPTARG
+        ;;
+    *)
+        echo "Incorrect options provided"
+        exit 1
+        ;;
+    esac
+done
+
```

Comment:
Insert this bit:
````
shift $(($OPTIND - 1))
tests="$@"
````
which takes all of the unprocessed args and shoves them into `tests`